src/AWS.Deploy.Orchestration/TemplateEngine.cs

```diff
@@ -67,7 +67,7 @@ public void GenerateCDKProjectFromTemplate(Recommendation recommendation, Orches
             var templateParameters = new Dictionary<string, string> {
                 // CDK Template projects can parameterize the version number of the AWS.Deploy.Recipes.CDK.Common package. This avoid
                 // projects having to be modified every time the package version is bumped.
-                { "AWSDeployRecipesCDKCommonVersion", FileVersionInfo.GetVersionInfo(typeof(Constants.CloudFormationIdentifier).Assembly.Location).ProductVersion
+                { "AWSDeployRecipesCDKCommonVersion", FileVersionInfo.GetVersionInfo(typeof(AWS.Deploy.Recipes.CDK.Common.CDKRecipeSetup).Assembly.Location).ProductVersion
```

Comment:
Since we dropped Nerdbank.GitVersioning, this code was bugged as it always returned 1.0.0. This is because `Constants.CloudFormationIdentifier` is part of `AWS.Deploy.Orchestration` which is not versioned. This is why I added a project reference to `AWS.Deploy.Recipes.CDK.Common` and used `AWS.Deploy.Recipes.CDK.Common.CDKRecipeSetup` which is versioned by `AutoVer`.